packages/paste-website/src/pages/inclusive-design/index.mdx

```diff
@@ -0,0 +1,409 @@
+          Use content-based, not device-specific, media queries{' '}
+          <Anchor showExternal href="http://bradfrost.com/blog/post/7-habits-of-highly-effective-media-queries/#content">
+            (Read more about why this matters)
+          </Anchor>
+        </ListItem>        
+        <ListItem>
+          Provide large touch "targets" for interactive elements{' '}
+          <Anchor showExternal href="http://www.bbc.co.uk/guidelines/futuremedia/accessibility/mobile/design/touch-target-size">
+            (Read more about why this matters)
+          </Anchor>
+        </ListItem>        
+        <ListItem>Ensure content is not obscured through zooming (no fixed widths)</ListItem>
+        <ListItem>Ensure the same content is available across different devices and platforms</ListItem>
+      </UnorderedList>
+    </DisclosureContent>
+  </Disclosure>
+  
+  <Disclosure>
+    <DisclosureHeading as="h3" variant="heading30">
+      Assistive technology considerations
+    </DisclosureHeading>
+    <DisclosureContent>
+      <UnorderedList>
+        <ListItem>
+          Use screen reader and keyboard accessible HTML{' '}
+          <Anchor showExternal href="https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML">(Read more about why this matters)</Anchor>
+        </ListItem>
+        <ListItem>Honor requests to remove animation via the `prefers-reduced-motion` media query</ListItem>
+        <ListItem>Do not recreate supported and expected browser behaviors with bespoke scripts</ListItem>
+        <ListItem>
+          Support Windows high contrast mode (use images, not background images){' '}
+          <Anchor showExternal href="http://adrianroselli.com/2012/08/css-background-images-high-contrast-mode.html">
+            (Read more about why this matters)
+          </Anchor>
+        </ListItem>
+        <ListItem>
+          Apply `alt="` or `aria-hidden="true"` to decorative images{' '}
+          <Anchor showExternal href="https://www.w3.org/WAI/tutorials/images/decorative/">(Read more about why this matters)</Anchor>
+        </ListItem>
+        <ListItem>Use data tables (`table`) for data only, not visual layout purposes</ListItem>
+        <ListItem>Make scrollable elements focusable for keyboard users</ListItem>
+        <ListItem>Ensure keyboard focus order is logical regarding visual layout</ListItem>
+        <ListItem>Do not hijack standard scrolling behavior</ListItem>
+        <ListItem>Move focus between dialogs and the controls that invoked them</ListItem>
+        <ListItem>Provide status and error messages as WAI-ARIA live regions</ListItem>
+        <ListItem>Provide clear, unambiguous focus styles</ListItem>
+        <ListItem>Ensure states (pressed, expanded, invalid, etc) are communicated to assistive software</ListItem>
+        <ListItem>Match semantics to behavior for assistive technology users</ListItem>
+        <ListItem>Ensure disabled controls are not focusable</ListItem>
```

Comment:
Our disabled tabs are focusable - is that an exception to this rule, or is that something we need to change?

Reply:
I don't like focusable disabled elements. It's a bug on Reakit. But equally, it's not a hill anyone should die on.